app/controllers/OnlineTestController.scala

```diff
@@ -62,10 +68,10 @@ trait OnlineTestController extends BaseController {
 
   import Commands.Implicits._
 
-  def getOnlineTest(userId: String) = Action.async { implicit request =>
-    onlineTestingService.getPhase1TestProfile(userId).map {
+  def getOnlineTest(applicationId: String) = Action.async { implicit request =>
```

Comment:
I guess this method could be renamed to getPhase1OnlineTests or getPhase1Tests. I made the same suggestion to ben for this method in another review.